webapp/packages/core-view/src/Action/Actions/ACTION_DELETE_ALL.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '../createAction.js';
+
+export const ACTION_DELETE_ALL = createAction('delete-all', {
+  label: 'ui_delete',
+});
```

Comment:
not sure about global delete all action cuz it seems specific (hard to imagine case when you need exactly delete all instead of delete)